com.unity.render-pipelines.high-definition/Runtime/Material/BuiltinGIUtilities.hlsl

```diff
@@ -222,8 +234,12 @@ float4 SampleShadowMask(float3 positionRWS, float2 uvStaticLightmap) // normalWS
 {
 #if defined(LIGHTMAP_ON)
     float2 uv = uvStaticLightmap * unity_LightmapST.xy + unity_LightmapST.zw;
-    float4 rawOcclusionMask = SAMPLE_TEXTURE2D(unity_ShadowMask, samplerunity_ShadowMask, uv); // Can't reuse sampler from Lightmap because with shader graph, the compile could optimize out the lightmaps if metal is 1
-#else
+    #ifdef UNITY_DOTS_INSTANCING_ENABLED
+        float4 rawOcclusionMask = SAMPLE_TEXTURE2D_ARRAY(unity_ShadowMasks, samplerunity_ShadowMasks_linear_clamp, uv, unity_LightmapIndex.x); // Can't reuse sampler from Lightmap because with shader graph, the compile could optimize out the lightmaps if metal is 1
```

Comment:
note: regarding my previous comment, you can reuse the same sampler here: samplerunity_ShadowMasks_linear_clamp instead of samplerunity_ShadowMask for the non dots

Reply:
Fixed locally